### DISCUSSION TITLE
VtM V5

### DISCUSSION THREAD

**Discussion by Vorkimarh** [silentjames/WoD-charsheets-for-Obsidian#8] [Q&A] [answered]
Доброго времени суток, хотел бы поинтересоваться, есть ли в планах делать чарники для 5ки? Во мне энтузиазма хватает самому сделать, если мне принцип объяснят))).

**Comment by silentjames** [comment, marked as answer]
Привет! 
Есть в очень отдаленных планах) Сам играю по двадцатке, и чарники тоже делаю для неё. 
С другой стороны, на текущей базе сделать будет не очень сложно, разве что с дисциплинами придется повозиться. Быть может, чуть позже займусь.
А если делать самому, то стоит поизучать мануал по основному плагину, Fantasy Statblock. Там в конструкторе можно создавать группы (вертикальные и горизонтальные), а в них уже прятать параметры (properties). Ну и немного пострадать с CSS :)

**Comment by Charax** [comment]
I would be really interested in this for v5 if you still want to work on it. Creating the statblock itself isn't too difficult but making it work with your beautiful CSS is very hard. Would you be willing to do this as a commission perhaps?

**Reply by silentjames** [reply]
Hey Charax, sorry for the super late reply!
I’d definitely be up for doing this as a commission if you’re still interested.
Let’s chat about the details on Discord.
